.github/workflows/tox-linters.yml

```diff
@@ -41,6 +41,11 @@ jobs:
         # - TOXENV: pre-commit-ci
         # - TOXENV: check-docs
         # - TOXENV: build-docs
+        exclude:
```

Comment:
Oh, I didn't know there was an `exclude:`. Where's the docs for it?

Reply:
Found it: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix.